src/Go/ui/GoSubnetSearch.tsx

```diff
@@ -24,16 +24,15 @@ export const GoSubnetSearch = ({ open, search, cancel, showInstructions }: IProp
     opponent === opponents.w0r1d_d43m0n ? 19 : Math.min(Player.go.boardState?.board?.[0]?.length ?? 7, 13);
   const [boardSize, setBoardSize] = useState(preselectedBoardSize);
 
-  const opponentFactions = [opponents.none, ...opponentList];
-  if (showWorldDemon()) {
-    opponentFactions.push(opponents.w0r1d_d43m0n);
-  }
+  const opponentFactions = getOpponentList(showWorldDemon(), true);
```

Comment:
I think this is the only place where you use the 2nd arg? I think it would be slightly clearer to just push the no opponent here.